gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,303 @@
+
+###################################
+NVIDIA GPU DRA Driver
+###################################
+
+*******************************
+About the NVIDIA GPU DRA Driver
+*******************************
+
+The NVIDIA GPU DRA Driver leverages the Kubernetes Dynamic Resource Allocation (DRA) API to support NVIDIA IMEX channels available in GH200 and GB200 GPUs.

```

Comment:
May be we should add a bit about imex channels. E.g. 
.... to support NVIDIA IMEX (Internode Memory Exchange/Management Service) channels available in GH200 and G200 systems that allows the GPUs to directly read / write each other’s memory over a high-bandwidth NVLink.

Reply:
defined in line 13 and 14